src/main/java/org/opentripplanner/middleware/tripmonitor/TrustedCompanion.java

```diff
@@ -42,10 +45,11 @@ private TrustedCompanion() {
         ConfigUtils.getConfigPropertyAsText("TRUSTED_COMPANION_CONFIRMATION_PAGE_URL");
     public static final String ACCEPT_KEY = "acceptKey";
     public static final String USER_LOCALE = "userLocale";
-    public static final String EMAIL_FIELD_NAME = "email";
+    public static final String EMAIL_PARAM = "email";
 
     /** Note: This path is excluded from security checks, see {@link OtpMiddlewareMain#initializeHttpEndpoints()}. */
     public static final String ACCEPT_DEPENDENT_PATH = "api/secure/user/acceptdependent";
+    public static final String GET_DEPENDENT_MOBILITY_PROFILE_PATH = "/:%s/getdependentmobilityprofile";
```

Comment:
I don't think we need to pass the user id here, it is being obtained from the token (I was by mistake leaving the placeholder as is, and the endpoint call proceeded normally).

Reply:
Updated: https://github.com/ibi-group/otp-middleware/pull/266/commits/3f5f7349fb1354b33c321ec510585e7fe9db65d5